lib/puppet_x/chocolatey/chocolatey_common.rb

```diff
@@ -0,0 +1,81 @@
+require 'pathname'
+require Pathname.new(__FILE__).dirname + 'chocolatey_version'
+require Pathname.new(__FILE__).dirname + 'chocolatey_install'
+
+module PuppetX
+  module Chocolatey
+    module ChocolateyCommon
+
+      def file_exists?(path)
+        File.exist?(path)
+      end
+      module_function :file_exists?
+
+      def chocolatey_command
+        if Puppet::Util::Platform.windows?
+          chocoInstallPath = PuppetX::Chocolatey::ChocolateyInstall.install_path
+          #default_location = $::choco_installpath || ENV['ALLUSERSPROFILE'] + '\chocolatey'
+          chocopath = ('C:\ProgramData\chocolatey' if file_exists?('C:\ProgramData\chocolatey\bin\choco.exe')) ||
```

Comment:
I'm not 100% comfortable with using a hardcoded path for C:\ProgramData.  It tends to be surfaced as `%ALLUSERSPROFILE%` or at least `%SYSTEMDRIVE%\ProgramData`